hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/8_8_0/8044-extend-transaction-log-support-to-batch-jobs.yaml

```
@@ -0,0 +1,8 @@
+---
+type: add
+issue: 8044
+title: "Add STORAGE_POSTSTORAGE_BATCH_JOB_CREATE and STORAGE_POSTCOMPLETE_BATCH_JOB pointcuts in batch 2 jobs.
+STORAGE_POSTSTORAGE_BATCH_JOB_CREATE is invoked after a batch job is persisted to the database and receives a
+ca.uhn.fhir.batch2.model.JobInstance that includes the job instance id as stored in the db
+STORAGE_POSTCOMPLETE_BATCH_JOB is invoked after the job instance is completed. JobInstance is immutable here and
+receives a ca.uhn.fhir.batch2.model.JobInstance"
```

Full class names like `ca.uhn.fhir.batch2.model.JobInstance` are unnecessary in changelog. Pointcut invocation details can be omitted as well, as they are described in pointcut javadoc. My suggestion:
```suggestion
title: "Two new pointcuts have been added to support transaction logging and monitoring for Batch2 jobs:
`STORAGE_POSTSTORAGE_BATCH_JOB_CREATE` and `STORAGE_POSTCOMPLETE_BATCH_JOB`."
```